README.adoc

```diff
@@ -1,88 +1,323 @@
+* Some quickstarts are based upon other quickstarts but have expanded capabilities and functionality. If a prerequisite quickstart is listed, make sure you deploy and test it before looking at the expanded version.
+
 
 [[run_the_quickstarts]]
-== Run the Quickstarts
+== Run the Quickstarts Using the Maven Command Line
 
 [[build_and_deploy_a_quickstart]]
 === Build and Deploy a Quickstart
 
-The root folder of each individual quickstart contains a `README{outfilesuffix}` file with detailed instructions on how to build and run the example. In most cases you do the following:
+The root folder of each individual quickstart contains a `README.html` file with detailed instructions on how to build and run the example. In most cases you do the following:
+
+* link:https://github.com/jboss-developer/jboss-developer-shared-resources/blob/master/guides/START_JBOSS_EAP.adoc#start_the_jboss_eap_server[Start the WildFly server].
+* Optionally, you can xref:build_the_quickstart_archive[build the quickstart archive] to test for compile errors.
+* You can also xref:verify_the_quickstarts_build_with_one_command[Verify that all of the quickstarts build] with one command.
+* xref:build_and_deploy_the_quickstarts[Build and deploy the quickstart].
+* xref:undeploy_an_archive[Undeploy the quickstart] when you are finished testing.
+* You can also xref:undeploy_the_deployed_quickstarts_with_one_command[undeploy the all of the deployed quickstarts] with one command.
+
+[[build_and_deploy_the_quickstarts]]
+=== Build and Deploy the Quickstarts
+
+See the `README` file in each individual quickstart folder for specific details and information on how to run and access the example.
+
+[[build_the_quickstart_archive]]
+==== Build the Quickstart Archive
+
+You can follow these steps to build the application to test for compile errors or to view the contents of the archive. See the specific quickstart `README` file for complete details.
+
+. Open a terminal and navigate to the root directory of the quickstart you want to build.
+. Use the following command if you only want to build the archive, but not deploy it.
++
+[source,options="nowrap"]
+----
+$ mvn clean install
+----
+
+[[build_and_deploy_the_quickstart]]
+==== Build and Deploy the Quickstart
+
+This section describes the basic steps to build and deploy an application. See the specific instructions in each quickstart `README` file for any variations to this process.
+
+. Make sure you start the WildFly server as described in the quickstart `README` file.
+. Open a terminal and navigate to the root directory of the quickstart you want to run.
+. Use the following command to build and deploy the archive.
++
+[source,options="nowrap"]
+----
+$ mvn clean install wildfly:deploy
+----
+
+[[undeploy_an_archive]]
+==== Undeploy an Archive
+
+Use the following command to undeploy the quickstart.
+
+[source,options="nowrap"]
+----
+$ mvn wildfly:undeploy
+----
+
+[[verify_the_quickstarts_build_with_one_command]]
```

Comment:
IMHO this kind of content is not for users, content wrt building/deploying/undeploying *all* QS at once  should be in a separated doc, focused on the QS project development.

Reply:
Agree. Removed.